lib/govuk_message_queue_consumer/test_helpers/test_consumer.rb

```diff
@@ -3,5 +3,13 @@ class TestConsumer < Consumer
     def publish_message(payload, options)
       exchange.publish(payload, options)
     end
+
+    # call after integration tests finish
+    def purge_and_unbind_queue(routing_key)
+      # purge all messages from the queue and
+      # unbind the queue with the given routing_key
+      queue.purge
+        .unbind(exchange, {routing_key: routing_key})
```

Comment:
Would it not be cleaner to just delete the queue?  Consumers are responsible for creating the queues in the first place, and setting up the routing key bindings, so that seems like the right thing to do to get back to the starting state.